games/s.yaml

```diff
@@ -402,6 +402,19 @@
     - http://ultrastardx.sourceforge.net/screenshots/game_visualisations.png
     - http://ultrastardx.sourceforge.net/screenshots/partymode_teams.png
 
+- name: [Snipes, Snipes (video game)]
+  remakes:
```

Comment:
This needs a name and yaml list member i.e.

    - name: Snipes
      remakes:
      - name: Snipes
        repo: ...